Gemfile.lock

```diff
@@ -42,6 +42,3 @@ DEPENDENCIES
   retriable (~> 2.0)
   run_loop!
   xcpretty
-
-BUNDLED WITH
-   1.10.6
```

Comment:
In general, don't commit the lock unless there is a update to the gem versions that you want.

I have not seen this message about BUNDLED WITH before.

This is non-blocking.